sdks/README.md

```diff
@@ -75,6 +79,24 @@ $ ./sidecar.linux.amd64 --local
 {"level":"info","msg":"Shutdown request has been received!","time":"2017-12-22T16:10:19-08:00"}
 ```
 
+### Writing your own SDK
+
+If there isn't a SDK for the language and platform you are looking for, you have several options:
+
+#### gRPC Client Generation
+
+If client generation is well supported by [gRPC](https://grpc.io/docs/), then generate a client from the
+[sdk.proto](../sdk.proto), and look at the current [sdks](.) to see how the wrappers are implemented to make interaction
+with the SDK server simpler for the user.
+
+#### REST API Implementation
+
+If client generation is not well supported by gRPC, or if there are other complicating factors, implement the SDK through
+the [REST](../docs/sdk_rest_api.md) HTTP+JSON interface. This could be written by hand, or potentially generated from
+the [Swagger/OpenAPI Spec](../sdk.swagger.json).  
```

Comment:
Should we show an example on how to do that ?

```
docker run --rm -v ${PWD}:/local swaggerapi/swagger-codegen-cli generate -i /local/sdks/sdk.swagger.json  -l cpprest -o /local/out/cpp
```

Do you plan to add a SDK using the http gateway ?

Reply:
Ooh. I like this idea. I'd like to point to a central documentation too - is this it?
https://swagger.io/docs/open-source-tools/swagger-codegen/

Re: to add an SDk using the gateway. I wasn't going to to do it with this PR, I figure Unreal + CPP would end up building on top of this work, and be the examples. We could likely follow the same pattern as the existing SDKs (light wrapper around the possibly generated client), but was waiting to see what an actual implementation would look like. WDYT?

Reply:
Yes that's the link, the plan sounds good !